src/pages/K8s.vue

```diff
@@ -7,7 +7,7 @@
     :notifications="notificationsList"
   >
     <div class="kubernetes-settings">
-      <labeled-input label="Kubernetes version">
+      <labeled-input :label="kubernetesVersionLabel()">
```

Comment:
My understanding was that, in Vue, methods with no arguments don't need the `()` — it'll be called automatically.  Is that incorrect? (That's quite likely; I'm more after the details of when that's incorrect)